### DISCUSSION TITLE
build packages/nocodb throw error

### DISCUSSION THREAD

**Discussion by tomorrowcheng** [nocodb/nocodb#3399] [General]
**Please enter the following details**

```
Node: **v16.17.0**
Arch: **x64**
Platform: **darwin**
Docker: **false**
Database: **sqlite3**
ProjectOnRootDB: **true**
RootDB: **sqlite3**
PackageVersion: **0.92.4**
```

**Steps To Reproduce**

1、cd packages/nocodb
2、npm install
3、npm run build

**Expected Behavior**

build success

**Screenshots**
throw error；I import https://github.com/faker-js/faker but npm run watch:run did not work
<img width="1229" alt="image" src="https://user-images.githubusercontent.com/42925114/187434188-d0d3694a-3e12-466a-acf4-6cf561e98f19.png">

**Comment by o1lab** [comment]
Is that installed ? Moving this to discussion.

**Reply by tomorrowcheng** [reply]
yes，it is installed
I find packages/nocodb/package.json don't has "faker", but KnexClient.ts required "faker"
